cli.js

```diff
@@ -195,7 +196,12 @@ var jobs = {
     logger.info(scope, 'generating route table');
     logger.verbose(scope, 'options %j', argv.route);
     try {
-      var rules = Minifier.minify(argv.route);
+      var rules;
+      if (argv.route.length == 2) {
+        rules = Minifier.minify(argv.route);
+      } else {
+        rules = MultiGwMinifier.minify(argv.route);
```

Comment:
It seems that this generalized the existing algorithm. Does it work with argv.route.length == 2 case as well?  If so, could you update code in minifier.js directly?